### ISSUE TITLE
Import Script Error on 1912 CU6

### ISSUE THREAD

**Issue by nobody-special06** [SLemonier/Citrix-Site-Import-Export#2]
Let me first start by saying AWESOME SCRIPT!

For some published apps the import script will error out stating "Specified application name is already in use" in the log.

When I check Studio no app by that name exist. Is there any way to bypass this error and keep the script going?
I have found a workaround to just delete the bad app section from the export.xml.

Also, the script will error for some apps stating the error below. The app still gets created in Studio but I have to keep excuting the script.

 "trying import
An error occured while adding a new PublishedApp"

**Comment by SLemonier** [comment]
thank you for your feedback :)
Could you share the export.xml file please (feel free to anonymise it)?
I must try to reproduce it.

**Comment by nobody-special06** [comment]
I can't provide the xml but it does look like we had some duplicates so I removed those and the script continued. 

I did run into another issue where the Export script didn't export all the configured apps in the site. Got the following warning in the message. Is there a flag to add MaxRecordCount to the script?

Enumerating Published Apps config...
WARNING: Only first 250 records returned. Use -MaxRecordCount to retrieve more.
